tools/respec2html.js

```diff
@@ -43,7 +48,9 @@ cli
   .option("-w, --haltonwarn", "Abort if ReSpec generates warnings.", false)
   .option("--disable-sandbox", "Disable Chromium sandboxing if needed.", false)
   .option("--devtools", "Enable debugging and show Chrome's DevTools.", false)
-  .option("--verbose", "Log processing status to stdout.", false);
+  .option("--verbose", "Log processing status to stdout.", false)
+  .option("--localhost", "Spin up a local server to peform processing.", false)
```

Comment:
Maybe we can support something like `--localhost=optionalPort` instead of two options.

Reply:
Could do... need to check how to set up that kind of option.

Reply:
Concluded to keeping option separate.